tests/baselines/reference/destructuringInFunctionType.js

```diff
@@ -42,19 +42,19 @@ interface c {
 declare type T1 = ([a, b, c]);
 declare type F1 = ([a, b, c]) => void;
 declare type T2 = ({
-    a;
+    a: any;

```

Comment:
This seems inconsistent with previous emit.

Reply:
As [above](https://github.com/Microsoft/TypeScript/pull/21930#discussion_r174663770), I reread the PR summary. It seems that we should have fixed `F1` as well.